docs/reference/rest-api/index.asciidoc

```diff
@@ -10,48 +10,46 @@ directly to configure and access {es} features.
 We are working on including more {es} APIs in this section. Some content might
 not be included yet.
 
-* <<docs>>
-* <<search>>
-* <<indices>>
-* <<cat>>
-* <<cluster>>
-* <<info-api,Info API>>
+* <<api-conventions, API conventions>>
+* <<cat, cat APIs>>
 * <<ccr-apis,{ccr-cap} APIs>>
+* <<cluster, Cluster APIs>>
 * <<data-frame-apis,{dataframe-cap} APIs>>
+* <<docs, Document APIs>>
 * <<graph-explore-api,Graph Explore API>>
-* <<indices-reload-analyzers,Reload Search Analyzers API>>
+* <<indices, Index APIs>>
 * <<index-lifecycle-management-api,Index lifecycle management APIs>>
-* <<snapshot-lifecycle-management-api,Snapshot lifecycle management APIs>>
+* <<info-api,Info API>>
 * <<licensing-apis,Licensing APIs>>
 * <<ml-apis,{ml-cap} {anomaly-detect} APIs>>
 * <<ml-df-analytics-apis,{ml-cap} {dfanalytics} APIs>>
+* <<migration-api,Migration APIs>>
+* <<indices-reload-analyzers,Reload Search Analyzers API>>
+* <<rollup-apis,Rollup APIs>>
+* <<search, Search APIs>>
 * <<security-api,Security APIs>>
+* <<snapshot-lifecycle-management-api,Snapshot lifecycle management APIs>>
 * <<watcher-api,Watcher APIs>>
-* <<rollup-apis,Rollup APIs>>
-* <<migration-api,Migration APIs>>
 --
 
-:leveloffset: +1
 include::{es-repo-dir}/api-conventions.asciidoc[]
-include::{es-repo-dir}/docs.asciidoc[]
-include::{es-repo-dir}/search.asciidoc[]
-include::{es-repo-dir}/indices.asciidoc[]
 include::{es-repo-dir}/cat.asciidoc[]
-include::{es-repo-dir}/cluster.asciidoc[]
-
-:leveloffset: -1
-include::info.asciidoc[]
 include::{es-repo-dir}/ccr/apis/ccr-apis.asciidoc[]
+include::{es-repo-dir}/cluster.asciidoc[]
```

Comment:
I think cluster.asciidoc should precede ccr-apis.asciidoc.  Otherwise they seem out of order:

![image](https://user-images.githubusercontent.com/26471269/61555874-3a42c400-aa15-11e9-854d-224e0c6e7425.png)

Reply:
Thanks for catching that. Fixed with a427817.